react/features/toolbar/reducer.js

```diff
@@ -0,0 +1,137 @@
+/* @flow */
+
+import { ReducerRegistry } from '../base/redux';
+
+import {
+    CLEAR_TOOLBAR_TIMEOUT,
+    SET_ALWAYS_VISIBLE_TOOLBAR,
+    SET_SUBJECT_SLIDE_IN,
+    SET_TOOLBAR_BUTTON,
+    SET_TOOLBAR_IS_HOVERED,
+    SET_TOOLBAR_TIMEOUT,
+    SET_TOOLBAR_TIMEOUT_NUMBER,
+    SET_TOOLBAR_VISIBILITY
+} from './actionTypes';
+import { getDefaultToolbarButtons } from './functions';
+
+declare var interfaceConfig: Object;
+
+/**
+ * Returns initial state for toolbar's part of Redux store.
+ *
+ * @returns {{
+ *     primaryToolbarButtons: Map,
+ *     secondaryToolbarButtons: Map
+ * }}
+ * @private
+ */
+function _getInitialState() {
+    // Default toolbar timeout for mobile app.
+    let toolbarTimeout = 5000;
+
+    if (typeof interfaceConfig !== 'undefined'
+            && interfaceConfig.INITIAL_TOOLBAR_TIMEOUT) {
+        toolbarTimeout = interfaceConfig.INITIAL_TOOLBAR_TIMEOUT;
+    }
+
+    return {
+        ...getDefaultToolbarButtons(),
+
+        alwaysVisible: false,
+        isHovered: false,
+        subjectSlideIn: false,
+        timeoutId: null,
+        toolbarTimeout,
+        visible: false
```

Comment:
1. Seriously, @BeatC, when I pointed out to you in this PR that our coding style requires jsdocs everywhere, I really meant that. You're defining properties above.

2. I find it inconsistent to use `boolean` variable naming with and without `is`. Please stick to one style. As we have `visible`, please rename `isHovered`, `isDock`, etc. to be consistent with `visible`.